isotovideo

```diff
@@ -262,6 +262,10 @@ unshift @INC, $bmwqemu::vars{CASEDIR} . '/lib';
 require $bmwqemu::vars{PRODUCTDIR} . "/main.pm";
 @INC = @oldINC;
 
+# for evaluation of the test schedule only governed by "main.pm" early exit
+# can be requested
+exit 0 if $bmwqemu::vars{EXIT_AFTER_SCHEDULE};
```

Comment:
Can you add the test for the full-stack test?

Reply:
I thought about it but hestitated to do so because the full stack test is already taking quite a long time, is hard to debug and encourages people to execute the tests even less likely on local instances and instead waste their time waiting for travis to report results after some time. *But* I added "t/14-isotovideo.t" for that purpose taking only 1 second to come back with a useful test result.

Reply:
Fair enough :), I wonder what's failing in travis... Just restarted the job...